de.metas.adempiere.adempiere/base/src/main/java/de/metas/report/ReportStarter.java

```diff
@@ -180,21 +180,8 @@ private void startProcessInvokeReportOnly(@NonNull final ReportPrintingInfo repo
 		//
 		// Set report data to process execution result
 		final ProcessExecutionResult processExecutionResult = processInfo.getResult();
-
+		final String reportFilename = result.getFilename() != null ? result.getFilename() : extractReportFilename(processInfo, outputType);
 		final String reportContentType = outputType.getContentType();
-
-		final String reportFilename;
```

Comment:
i kindof liked it this way (this version is also the fixed one). maybe it makes sense to extract it to a method like `extractResultFileName(result)`, but i particularly like the logging, for support purposes

Reply:
makes sense to have logging, will revert to the old way! (Also this got changed when merging the old branch)